v6_carrier_py/encryption.py

```diff
@@ -0,0 +1,71 @@
+from base64 import b64encode
+from hashlib import sha512
+from typing import List
+
+import pandas as pd
+
+
+def fixed_length_string(string: str, desired_length: int = 128):
+    """
+    Either truncate or padd a string with 'a' so that it is of the specified
+    length.
+    TODO: This is token from Chang's repo. Do we really want to support this or
```

Comment:
@dsmits what do you think?

Reply:
Yeah this is one of those details from changs repo that I think isn't totally necessary to implement. Let's enforce a 128 length salt, and make sure they are randomly generated when we create them.

Reply:
You don't really use it at the moment do you?

Reply:
Ok, I'll take it out :)